sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -913,7 +913,7 @@ object RapidsConf {
   val ENABLE_READ_CSV_DOUBLES = conf("spark.rapids.sql.csv.read.double.enabled")
     .doc("CSV reading is not 100% compatible when reading doubles.")
     .booleanConf
-    .createWithDefault(false)
+    .createWithDefault(true)
```

Comment:
do we need to update any tests?  ie tests assumed false so now we only test true

Reply:
The only place to set it explicitly is https://github.com/NVIDIA/spark-rapids/blob/branch-22.08/integration_tests/src/main/python/csv_test.py#L171.
Every reading test related to double set spark.rapids.sql.csv.read.double.enabled to true explicitly.
I don't find any test case which requires it be false.

Reply:
Same as json_test.py.

Reply:
@tgravescs  Can you review it again?